lib/teracy-dev/location/git_synch.rb

```diff
@@ -51,11 +59,11 @@ def sync(location, sync_existing)
         dir = git_config['dir']
 
         if File.exist? path
+          updated = update_remote(path, git_remote)
```

Comment:
when remote repos are updated, we don't consider this as sync updated, so let's not trigger sync update for it.

sync updated should be true only when there are actual changes of code.